src/components/ItemPicker/ItemPicker.vue

```diff
@@ -270,10 +272,6 @@ export default {
       return this.disabled || this.disabledSorting
     },
 
-    /**
```

Comment:
why remove this?

Reply:
thought this comment was for me; adding it back